frontend/src/components/events/atoms/flex/flex.css

```diff
@@ -0,0 +1,32 @@
+.flex {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin: 20px;
+}
+
+.flex-item {
+  display: flex;
+  flex: 0 1 27%;
```

Comment:
this can just be `flex: 1` -> as it will allow flex to just handle the rest of the free space by it self -> https://stackoverflow.com/questions/37386244/what-does-flex-1-mean/42630660

this will allow you to remove all of the mobile styles below.